code/modules/paperwork/photography.dm

```diff
@@ -31,16 +31,22 @@ var/global/photo_count = 0
 	randpixel = 10
 	w_class = ITEM_SIZE_TINY
 	var/id
-	var/icon/img	//Big photo image
-	var/scribble	//Scribble on the back.
+	///Big photo image
+	var/icon/img
+	///Scribble on the back.
+	var/scribble
 	var/image/tiny
 	var/photo_size = 3
-	var/anomalous = FALSE //ADD SUPPORT FOR MEMETICS/096/OTHERS TO SPREAD THROUGH PHOTOS
-	var/anomalytype //This has to inherit from the camera
-	var/anomalymob //This has to inherit from the camera
+	///Photo data containing mobs and objects.
+	var/list/meta_data
```

Comment:
isn't it holding regs?

Reply:
fixed to use weakrefs (and updated description)